solvation_analysis/rdf_parser.py

```diff
@@ -39,10 +91,28 @@ def plot_interpolation_fit(bins, rdf, **kwargs):
 
 
 def good_cutoff(f, cutoff_region, cr_pts, cr_vals):
+    """
+    Identifies whether or not the interpolation method has identified a valid
+    solvation cutoff. This fail if there is no solvation shell.
+
+    Parameters
+    ----------
+    f : an interpolated rdf function
+    cutoff_region : tuple
+        boundaries in which to search for a solvation shell cutoff, i.e. (1.5, 4)
+    cr_pts : np.array
+        the x-axis value of the extrema
+    cr_vals : np.array
+        the y-axis values of the extrema
+    Returns
+    -------
+    boolean : True if good cutoff, False if bad cutoff
+
+    """
     if (
-        len(cr_pts) < 2  # insufficient critical points
-        or cr_vals[0] < cr_vals[1]  # not a max and min
-        or not (cutoff_region[1] < cr_pts[1] < cutoff_region[4])  # min not in cutoff
+            len(cr_pts) < 2  # insufficient critical points
+            or cr_vals[0] < cr_vals[1]  # not a max and min
+            or not (cutoff_region[1] < cr_pts[1] < cutoff_region[4])  # min not in cutoff
```

Comment:
Docs say that `cutoff_region` is a tuple with an example of length 2 given, here we access the 5th element with `cutoff[4]`. Am I missing something here?

Reply:
Nope! That was a typo that I would have caught had I been doing TDD better :)

Fixed it and added testing.